src/scss/_index.scss

```diff
@@ -0,0 +1,22 @@
+hr{
+    border-top: 1px solid #000 !important;
```

Comment:
`!important` is a sign that Bootstrap is not being overriden properly. It contains styles for `hr` elements that you can override!

Reply:
Took away the !important and change the color to $pink.